app/controllers/iiif_controller.rb

```diff
@@ -71,7 +71,7 @@ def set_image_response_headers
   end
 
   def set_attachment_content_disposition_header
-    response.headers['Content-Disposition'] = "attachment;filename=#{params[:identifier]}.#{params[:format]}"
+    response.headers['Content-Disposition'] = "attachment;filename=#{identifier_params[:file_name]}.#{params[:format]}"
```

Comment:
Hmm, okay, so the only question I have is are are we 100% sure all the identifiers will always include the slash (encoded as it is now)?  If it does not included the slash, we'll pass `nil` here (which I'm not exactly sure what would be the end-user experience there).

That could be a totally baseless concern, so I'm fine with :ship:ing and fixing if we find that we do have identifiers that do not include the `/`.